File: Templete/bar_chart.R
library('ggplot2')

df_used_cars <- read.csv("/Users/lindazhong/Documents/Data science 365/Data Visualization/car_brand.csv")

bar_chart <- ggplot(df_used_cars,
                    aes(x = Brand, y = Cars.Listings))+
                    geom_bar(stat = 'identity', width = 0.8,color = 'navy', fill = 'navy')+
                    ggtitle("Car Listings by Brand")+
                    theme_classic()+
                    theme(axis.text.x = element_text(angle = 45, hjust = 1))+
                    labs(x = NULL,
                         y = "Number of Listings")
                  
bar_chart

File: Templete/Python/Bar_chart.py
# %%
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
# %%
df_used_cars = pd.read_csv(r'/Users/lindazhong/Documents/Data science 365/Data Visualization/car_brand.csv')
df_used_cars
# %%
sns.set()
plt.figure(figsize=(9,6),dpi=100)
plt.title("Cars Listings by Brand", fontweight= "bold", fontsize= 16)
plt.ylabel("Number of Listings", fontsize = 13)
plt.bar(x = df_used_cars["Brand"], height = df_used_cars["Cars Listings"],color = ["r","g","b","w","y","m","c"])
plt.xticks(rotation= 45, fontsize=13)
plt.yticks(fontsize=13)
plt.savefig("Use Cars Bar.png");
# %%
sns.set()
plt.figure(figsize=(9,6), dpi = 100)
plt.bar(x = df_used_cars["Brand"], height = df_used_cars["Cars Listings"], color= "midnightblue")
plt.xticks(rotation = 45,fontsize=13)
plt.yticks(fontsize=13)
plt.title("Car Listings by Brand", fontsize=16, fontweight="bold")
plt.ylabel("Number of Listings",size=13);
